tests/integration/conftest.py

```
@@ -1,16 +1,31 @@
+- Safe imports that don't fail when core dependencies are missing
+- VCR configuration for recording API responses
+- Rate limiting between tests
+- Test fixtures for common data
+"""
 import logging
 import os
 import re
 import time
 from collections.abc import Generator
 from pathlib import Path
+from typing import Any
```

Ruff (0.11.9)

`typing.Any` imported but unused

Remove unused import: `typing.Any`

(F401)